src/core.c

```diff
@@ -4297,11 +4295,25 @@ static bool InitGraphicsDevice(int width, int height)
     }
 #endif
 
+    CORE.Window.currentFbo.width = fbWidth;
+    CORE.Window.currentFbo.height = fbHeight;
+    CORE.Window.render.width = CORE.Window.currentFbo.width;
+    CORE.Window.render.height = CORE.Window.currentFbo.height;
```

Comment:
These four lines are the most important part. Before this change, `CORE.Window.currentFbo` and `CORE.Window.render.width` were filed with the `width` and `height` passed in parameter by the user to this function. It was not correct because when creating a window at a specific size, glfw create a framebuffer with the size `window size * dpi scale factor` (for all desktop platform except macos).